nvbench/main.cuh

```diff
@@ -54,6 +54,12 @@
 // clang-format on
 #endif
 
+#ifndef NVBENCH_ENVIRONMENT
+struct no_environment
+{};
```

Comment:
should we prefer to have`struct no_environment` in global namespace or `nvbench` namespace?

Reply:
That is a good suggestion. I think it should be in the `nvbench` namespace.